app/src/pages/CollectionPage/CollectionPage.tsx

```diff
@@ -49,7 +50,7 @@ export const CollectionPage = (): React.ReactElement => {
             </LayerContainer>
           </Box>
           <ContainingView>
-            <Stack direction={Direction.Vertical} isFullWidth={true} childAlignment={Alignment.Center} padding={PaddingSize.Wide2}>
+            <Stack direction={Direction.Vertical} isFullWidth={true} childAlignment={Alignment.Center}>
```

Comment:
this padding seemed fine to me, why did you remove it? can you update the screenshots after changes pls.